docs/units/equivalencies.rst

```diff
@@ -347,6 +347,39 @@ and you want to know how big your pixels need to be to cover half an arcsecond::
     >>> (0.5*u.arcsec).to(u.micron, tel_platescale)  # doctest: +FLOAT_CMP
     <Quantity 18.9077335632719 micron>
 
+Reduced hubble constant/"little-h" Equivalencies
```

Comment:
hubble -> Hubble
Equivalencies -> equivalencies

Reply:
I was once told I had a "Case of Random Capitalization disorder".  It appears I've had a flare-up...